lib/system/nimscript.nim

```diff
@@ -379,11 +379,26 @@ when not defined(nimble):
   template `==?`(a, b: string): bool = cmpIgnoreStyle(a, b) == 0
   template task*(name: untyped; description: string; body: untyped): untyped =
     ## Defines a task. Hidden tasks are supported via an empty description.
+    ##
     ## Example:
     ##
     ## .. code-block:: nim
     ##  task build, "default build is via the C backend":
     ##    setCommand "c"
+    ##
+    ## For a task named ``foo``, this template generates a ``proc`` named
+    ## ``fooTask``.  This is useful if you need to call one task in
+    ## another in your Nimscript.
+    ##
+    ## Example:
+    ##
+    ## .. code-block:: nim      # > nim foo
```

Comment:
So you commented on the ``.. code-block`` itself? Come on, did you even check how this renders in the HTML?

Reply:
No .. let me check. A comment breaks that? Is that a bug?

Reply:
Now, fixed and confirmed as well.

![image](https://user-images.githubusercontent.com/3578197/80865749-630d1080-8c59-11ea-8f97-ea180fef2e61.png)